utilities/pom.xml

```diff
@@ -218,11 +219,17 @@
 			<groupId>com.google.apis</groupId>
 			<artifactId>google-api-services-gmail</artifactId>
 		</dependency>
+
 		<dependency>
 			<groupId>com.google.apis</groupId>
 			<artifactId>google-api-services-calendar</artifactId>
 		</dependency>
 
+        <dependency>
+            <groupId>com.codeborne</groupId>
+            <artifactId>selenide</artifactId>
+        </dependency>
+
```

Comment:
i'm not sure if including `selenide` in the utilities is the right choice

Reply:
by the way, why? :dancer: it is not used anywhere

Reply:
its just a test, not final PR

Reply:
I was refreshing selenide in many places to observe change of behavior locally, that was just leftover. PR is now ready, tests should PASS.